packages/terra-clinical-data-grid/src/subcomponents/SectionHeader.jsx

```diff
@@ -68,7 +67,7 @@ class SectionHeader extends React.Component {
   }
 
   handleKeyDown(event) {
-    if (event.nativeEvent.keyCode === KEYCODES.ENTER || event.nativeEvent.keyCode === KEYCODES.SPACE) {
+    if (event.nativeEvent.keyCode === KeyCode.KEY_ENTER || event.nativeEvent.keyCode === KeyCode.KEY_SPACE) {
```

Comment:
Should be KEY_RETURN here too.

Reply:
Here is my fix. https://github.com/cerner/terra-clinical/pull/479/commits/5fad38da91527764167715a3cfc8bbe5174bc3b5#diff-50ba9af42cf24cd2956bd5b85e8df39bR70